met-web/src/components/common/Modals/types.tsx

```diff
@@ -1,6 +1,11 @@
+export interface ModalSubtext {
+    text: string;
+    bold: boolean;
```

Comment:
bold should be an optional field bold?: string. And we don't have to do bold: false when we create a ModalSubtext, we only pass it if it's true